cmd/shell.go

```diff
@@ -302,3 +302,13 @@ func DebugServer(addr string) {
 		log.Fatalf("unable to boot debug server: %v", err)
 	}
 }
+
+// ReadJSONFile takes a file path as an argument and attempts to
+// unmarshal the content of the file into a struct containing a
+// configuration of a boulder component.
+func ReadJSONFile(filename string, out interface{}) {
```

Comment:
This function should return an error rather than calling `FailOnError`. Then the caller can call `FailOnError` if appropriate.